cli/wallet/multisig.go

```diff
@@ -80,14 +86,18 @@ func signStoredTransaction(ctx *cli.Context) error {
 		if err != nil {
 			return cli.NewExitError(fmt.Errorf("failed to create RPC client: %w", err), 1)
 		}
-		res, err := c.SendRawTransaction(tx)
+		res, err = c.SendRawTransaction(tx)
 		if err != nil {
 			return cli.NewExitError(fmt.Errorf("failed to submit transaction to RPC node: %w", err), 1)
 		}
-		fmt.Fprintln(ctx.App.Writer, res.StringLE())
-		return nil
+		if ctx.Bool("await") {
+			version, err := c.GetVersion()
+			aer, err = waiter.New(c, version).Wait(res, tx.ValidUntilBlock, err)
+			if err != nil {
+				return cli.NewExitError(fmt.Errorf("the transaction doesn't accepted to the block: %w", err), 1)
+			}
+		}
 	}
-
-	fmt.Fprintln(ctx.App.Writer, tx.Hash().StringLE())
+	txctx.DumpTransactionInfo(ctx.App.Writer, res, aer)
```

Comment:
OK, my bad, we can't use `res` here because rpc not always specified and thus, res may be empty here. Sorry for misleading.